core/config_test.go

```diff
@@ -550,26 +550,36 @@ func TestChainConfigGetSet(t *testing.T) {
 	}
 }
 
-func TestChainConfig_GetLastRequiredHashFork(t *testing.T) {
+func TestChainConfig_GetLatestRequiredHashFork(t *testing.T) {
```

Comment:
Updated test for immediately above, PR https://github.com/ethereumproject/go-ethereum/pull/612.